packages/marshal/src/types.js

```diff
@@ -4,16 +4,16 @@ export {};
 /**
  * @template Slot
  * @callback ConvertValToSlot
- * @param {import('@endo/pass-style').PassableCap} val
+ * @param {any} val a PassableCap
```

Comment:
That seems like a terrible regression, possibly due to types not being built!

Reply:
It's not a regression. `PassableCap` in master is `any`. 
<img width="504" alt="Screenshot 2024-01-08 at 1 10 49 PM" src="https://github.com/endojs/endo/assets/21505/411222c1-d850-4048-9a8b-436ccf300574">

Reply:
Then the issue is with the `PassableCap` definition, not this param definition.

Reply:
There was indeed an issue with the PassableCap definition. It was `any`. The type is still `any` so there's no regression here.

Once we fix those types, I expect to return to this.

Reply:
Fix in https://github.com/endojs/endo/pull/1933